### ISSUE TITLE
Error: ENOENT: no such file or directory, open 'libc++abi.so.1'

### ISSUE THREAD

**Issue by probonopd** [KhushrajRathod/TeleDrive#16]
Thi @KhushrajRathod, thanks for making an AppImage. We'd like to add it to https://appimage.github.io/. However, it does not run on Xubuntu 18.04:

```
Error: ENOENT: no such file or directory, open 'libc++abi.so.1'
```

This is most likely because the library is not bundled inside the AppImage.

Reference:
https://github.com/AppImage/appimage.github.io/pull/2289

**Comment by KhushrajRathod** [comment]
Hi,
Thanks for reporting this. I'll fix this tomorrow as there are probably more dependencies missing on Xubuntu so I'll have to setup a VM.
It's nice to know someone noticed my project before I told anyone 😄

**Comment by KhushrajRathod** [comment]
Fixed in [v0.9.8](https://github.com/KhushrajRathod/TeleDrive/releases/tag/v0.9.8)

**Comment by probonopd** [comment]
Thank you very much, it now passes the test :+1:
